packages/cli/src/lib/snapshot/pullModel/validation/model.ts

```diff
@@ -0,0 +1,33 @@
+import pullModelSchema from './model.schema.json';
+import {ResourceSnapshotType} from '@coveord/platform-client';
+import {SnapshotPullModelResources} from '../interfaces';
+
+// TODO: not sure this is necessary
+export function getSnapshotModel() {
```

Comment:
This method will augment the schema with the available resources from the platform-client.
I don't know if this is necessary as the resource validation is already being done in the backend. The error message from the API is pretty clear if the SPM contains an invalid resource type.